contrib/update-storage-times.py

```diff
@@ -170,6 +172,11 @@ def cli_opts():
     parser.add_argument('--aggregate', action='store_true', dest='aggregate',
                         help="Passed through to whisper-resize.py, roll up values",
                         default=False)
+    parser.add_argument('--bindir', action='store', dest='bindir',
+                        help="The root path to whisper-resize.py and whisper-info.py",
+                        default='/opt/graphite/bin')
+    parser.add_argument('--sleep', action='store', dest='sleep',
+                        help="Sleep this amount of time in seconds between metric comparisons")
```

Comment:
Think you accidentally added this sleep argument which doesn't do anything. Could you please remove the argument, or add the feature if you have it ready?

Reply:
Good catch, that was something I was thinking about adding but hadn't completed. Should be good now though.